vllm/spec_decode/draft_model_runner.py

```diff
@@ -12,21 +22,10 @@
         # vllm_flash_attn is not installed, try the ROCm FA metadata
         from vllm.attention.backends.rocm_flash_attn import (
             ROCmFlashAttentionMetadata as FlashAttentionMetadata)
-except (ModuleNotFoundError, ImportError) as err:
-    raise RuntimeError(
+except Exception as e:
```

Comment:
you are changing behavior here from terminating to continuing with warning

Reply:
Hi, @michalkuligowski , yes, the spec_decode_worker.py imports draft_model_runner.py. And above ModuleNotFoundError in draft_model_runner.py will cause termination. That is why I changed the beviour to avoid termination. 

If this is not preferred, I can move the `from vllm.spec_decode.draft_model_runner import TP1DraftModelRunner` into `SpeDecoderWorker.create_worker` as below to avoid unwanted importError termination. => Will that be more acceptable to you?

```
else:
                if draft_tp == 1:
                    from vllm.spec_decode.draft_model_runner import TP1DraftModelRunner
                    draft_worker_kwargs[
                        "model_runner_cls"] = TP1DraftModelRunner
                else:
                    if draft_worker_kwargs[
                            "model_config"].hf_config.model_type == "eagle":
                        raise NotImplementedError(
                            "EAGLE does not support TP > 1 yet")
```

Reply:
Fixed above comments by check with platform. and only raise when platform is not one of Cuda, Roc or HPU.